### PR TITLE
Create popup modal

### PR DESCRIPTION
**Pull request by Lornakaboro**
## In this activity we did the following :

- Stored all the work-section data in a JavaScript object
- Used Java Script to display this section dynamically
- Created a pop-up section for both mobile and desktop versions dynamically using JavaScript
- Declared a function so that when the see project button is clicked the popup section is displayed and when the X-icon is clicked the pop-up section is closed.
- Looped through the JavaScript object so that  the pop-up displays the image of the corresponding card

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,18 +1,296 @@
...
+            <h2 class='work-title'>${projectData[index].name}</h2>
+            <i class='fa-regular fa-x mobile-popup-x-icon' onclick='closePopUp()'></i>
+        </div>
+        <div class="work-pointers">
+          <h3>CANOPY</h3>
+          <div class="work-pointers-bullets"></div>
+          <p>Back End Dev</p>
+          <div class="work-pointers-bullets"></div>
+          <p>2015</p>
+        </div>
```
Comment (thecodechaser) [inline script.js:133]: - [ ] For the desktop screens, Please reduce the gap between tags and make sure the design matches the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=G8H15csQc9GZNQtT-0) design.

### Your design
image

###  Figma design
image

File: script.js [hunk 2]
```diff
@@ -1,18 +1,296 @@
...
+        <div class='mobile-popup-links'>
+            <a href='${projectData[index].linkToLiveVersion}' class='mobile-popup-project-link mobile-popup-project-link1'>
+                See live
+                <img src='/images/Popup-icon.png' alt='image of a small icon' class='popup-icon'>
+            </a>
+            <a href='${projectData[index].linkToSource}' class='mobile-popup-project-link'>
+                See Source
+                <img src='/images/github-vector.png' alt='github icon' class='popup-icon'>
+            </a>
+        </div>
```
Comment (thecodechaser) [inline script.js:148]: - [ ] For the desktop screens, Please increase the gap between these buttons and reduce the gap between the button's text and icon to match the Figma design.

### Your design
image

### Figma design
image

Comment (thecodechaser) [inline script.js:148]: - [ ] For the mobile screens, Please reduce the button width to match with the Figma design.

### Your design
image

### Figma design
image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Lornakaboro, and @shegy28👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 2 above)
Comment (GzimAsani) [reply script.js:148]: - [ ] This requirement from the previous reviewer is not yet implemented
image

Comment (GzimAsani) [reply script.js:148]: - [ ] This requirement from the previous reviewer is not yet implemented
image

**Comment by Lornakaboro** [comment]
I don't understand what you mean..I have implemented the changes on the
bullets kindly elaborate

**Comment by GzimAsani** [comment]
- [ ] I meant this one, sorry adding two pictures by mistake
image

File: script.js [hunk 3]
```diff
@@ -1,18 +1,297 @@
...
+        <img src='${projectData[index].featuredImage}' class='work-image' alt='image of a lady doing yoga'>
+        <p class='work-description'>${projectData[index].description}</p>
+        <ul class='tags'>
+            ${technologiesUsed}
+        </ul>
+        <div class='mobile-popup-links'>
+            <a href='${projectData[index].linkToLiveVersion}' class='mobile-popup-project-link mobile-popup-project-link1'>
+                See live
+                <img src='/images/Popup-icon.png' alt='image of a small icon' class='popup-icon'>
```
Comment (BregornOriginal) [inline script.js:142]: [OPTIONAL] it seems that this icon it's not being loaded. You could try another one, and also to this works you should add a `.` before / so it looks like this `'./images/Popup-icon.png'`

File: script.js [hunk 4]
```diff
@@ -1,18 +1,297 @@
...
+            </a>
+            <a href='${projectData[index].linkToSource}' class='mobile-popup-project-link'>
+                See Source
+                <img src='/images/github-vector.png' alt='github icon' class='popup-icon'>
```
Comment (BregornOriginal) [inline script.js:146]: [OPTIONAL] it seems that this icon it's not being loaded. 
To this make work change the path, add a `.` before / so it looks like this `'./images/popup-icon.png'`

**Review by BregornOriginal — APPROVED** [review]
## Hi @Lornakaboro 💯 💯 💯 

## Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳 🥳 🥳 

## Congratulations!

## Highlights

✅ Linters are passing
✅ Correct use of GitHub Flow
✅ Good commit history
✅ Good work implementing the suggested changes from the previous reviewer! 🥇 
✅ README looks professional 🥇 

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

i-approve-gif-10


Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**